src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is doing its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working.");

```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should be more informative. Specify that a `Bulldozer` has stopped working.